controllers/cards.js

```diff
@@ -15,16 +15,23 @@ module.exports.createCard = (req, res) => {
       if (err.name === 'ValidationError') {
         res.status(400).send({ message: 'Некорректное название карточки или неверная ссылка на ресурс' });
       } else {
-        res.status(500).send({ message: err.message });
+        res.status(500).send({ message: 'Произошла ошибка' });
       }
     });
 };
 
 module.exports.deleteCard = (req, res) => {
-  Card.findByIdAndRemove(req.params.cardId)
+  Card.findById(req.params.cardId)
     .orFail(() => new NotFoundError('Нет карточки с таким id'))
-    .then((card) => res.send({ data: card }))
-    .catch((err) => res.status(err.statusCode || 500).send({ message: err.message }));
+    .then(async (card) => {
+      await Card.deleteOne(card);
+      res.send({ data: card });
+    })
```

Comment:
Можно лучше: использование `async/await` выглядит лишним.
Эта связка позволяет писать асинхронный код в синхронном виде. Но, когда большая часть кода написана через _chainable_ функции, то выглядит немного не к месту. Рекомендую продолжить с промисами:
```suggestion
    .then((card) => Card.deleteOne(card).then(() => res.send({ data: card })))
```